tests/safeds_stubgen/stubs_generator/__snapshots__/test_generate_stubs/test_type_var_creation.sdsstub

```diff
@@ -0,0 +1,17 @@
+@PythonModule("various_modules_package.type_var_module")
+package variousModulesPackage.typeVarModule
+
+@Pure
+@PythonName("type_var_func")
+fun typeVarFunc<_type_var>(
```

Comment:
Type parameter names should be `UpperCamelCase`. `@PythonName` is not needed for them, but the names must be replaced consistently.